### DISCUSSION TITLE
MobX-State-Tree wants to work more closely with y'all!

### DISCUSSION THREAD

**Discussion by coolsoftwaretyler** [mobxjs/mobx#3743] [General]
Hey folks,

I recently joined [MobX-State-Tree](https://github.com/mobxjs/mobx-state-tree) as a maintainer, alongside @jamonholmgren.

We've had a handful of maintainer meetings over the past few months, and we're working to re-invigorate our library. We care quite a bit about the MST community and our downstream users, but realized that neither of us are in close contact with MobX.

We'd like to change that, and be better partners in the overall ecosystem.

Is there anyone here that's particularly interested in chatting with us? I think we'd like to start by finding 20-30 minutes to say hi, introduce ourselves, and informally put names to faces. Afterwards, we'd love to figure out ways we can support MobX in our work on MST, and perhaps find some regular cadence of touchpoints for the two teams.

Let me know, please! Thanks in advance!

**Comment by mweststrate** [comment]
Happy to chime in :)

**Reply by coolsoftwaretyler** [reply]
Hey @mweststrate! That's awesome to hear - we weren't sure if you specifically would have much time for MST these days.

Should we invite you to the upcoming maintainer meeting? Is there anyone else here or that you can think of who wants to hop on as well?

**Reply by jamonholmgren** [reply]
Bump @mweststrate ... and anyone else?
